core/templates/dev/head/pages/header_js_libs.html

```diff
@@ -4,22 +4,30 @@
 <!--[if gte IE 9]><!-->
 <!-- jquery.js, angular.js and jquery-ui.js are removed from bundled js because they need to be at the header. Including
 bundled js at the header will block rendering.-->
+
   <script src="/third_party/static/jquery-2.1.1/jquery.min.js">
   </script>
 <!--<![endif]-->
-<script src="/third_party/static/jqueryui-1.10.3/jquery-ui.min.js">
-</script>
-<script src="/third_party/static/angularjs-1.4.7/angular.min.js"></script>
-<script src="/third_party/static/jquery-ui-touch-punch-0.3.1/jquery.ui.touch-punch-improved.js"></script>
+  <script src="/third_party/static/jqueryui-1.10.3/jquery-ui.min.js">
```

Comment:
No -- lines 11-16 should be indented with script flush-left. The reason lines 8 and 9 are indented is because they're in an if/endif construct (for IE browsers).

It's only lines 17 - 21 that should be indented further, because they are inside a script tag. (Similar to lines 24-31, before.) The remaining lines should be left as-is.